src/controllers/recipes_controller.js

```diff
@@ -1,97 +1,100 @@
 
-    pythonProcess.stdout.on("data", (data) => {
-        res.status(200).json(data.toString());
-    });
+  pythonProcess.stdout.on('data', (data) => {
+    res.status(200).json(data.toString());
+  });
 };
 
-module.exports.selectRecipeById = ({params: {recipeId = null}}, res) => {
-    if (recipeId === null) {
-        return res.status(400).json("Please enter the recipe Id");
-    }
+module.exports.selectRecipeById = ({ params: { recipeId = null } }, res) => {
+  if (recipeId === null) {
+    return res.status(400).json('Please enter the recipe Id');
+  }
 
-    recipesModel.selectRecipeById(recipeId, (err, result) => {
-        res.status(err ? 500 : 200).json(err ? undefined : result);
-    });
+  return recipesModel.selectRecipeById(recipeId, (err, result) => {
+    res.status(err ? 500 : 200).json(err ? undefined : result);
+  });
 };
 
 module.exports.processRecipesJson = (req, res) => {
-    const rawRecipeData = fs.readFileSync("data/recipes/recipes.json");
-    const recipes = JSON.parse(rawRecipeData);
+  const rawRecipeData = fs.readFileSync('data/recipes/recipes.json');
+  const recipes = JSON.parse(rawRecipeData);
 
-    const ingredientsList = recipes.map(({id, ingredients}) => ({
-        id,
-        ingredients: ingredients.map((ingredient) => ingredient.ingredientID)
-    }));
+  const ingredientsList = recipes.map(({ id, ingredients }) => ({
+    id,
+    ingredients: ingredients.map(ingredient => ingredient.ingredientID),
+  }));
 
 
-    const pythonProcess = spawn("python", ["recommender.py", PYTHON_MODES.PROCESS]);
+  const pythonProcess = spawn('python', ['recommender.py', PYTHON_MODES.PROCESS]);
 
-    pythonProcess.stdin.write(JSON.stringify(ingredientsList));
-    pythonProcess.stdin.end();
+  pythonProcess.stdin.write(JSON.stringify(ingredientsList));
+  pythonProcess.stdin.end();
 
-    let dataString = "";
+  let dataString = '';
 
-    pythonProcess.stdout.on("data", (data) => {
-        dataString += data;
-    });
+  pythonProcess.stdout.on('data', (data) => {
+    dataString += data;
+  });
 
-    pythonProcess.stderr.on("data", (error) => {
-        console.error(error.toString());
-    });
+  pythonProcess.stderr.on('data', (error) => {
+    console.error(error.toString());
+  });
 
-    pythonProcess.stdout.on("end", () => {
-        res.status(200).json(dataString.toString());
-    });
+  pythonProcess.stdout.on('end', () => {
+    res.status(200).json(dataString.toString());
+  });
 };
 
-module.exports.recommendRecipe = ({params: {recipeId = null}}, res) => {
-    if (recipeId === null) {
-        return res.status(400).json("Please enter the recipe Id");
-    }
 
-    const pythonProcess = spawn("python", ["recommender.py", PYTHON_MODES.RECOMMEND, recipeId]);
+module.exports.recommendRecipe = ({ params: { recipeId = null } }, res) => {
+  if (recipeId === null) {
+    return res.status(400).json('Please enter the recipe Id');
+  }
+
+  const pythonProcess = spawn('python', ['recommender.py',
```

Comment:
if we're defining an array on separate lines, could the first element also be on a separate line?

Reply:
yup, will change!